src/main/java/seedu/address/model/person/Address.java

```diff
@@ -26,20 +19,12 @@ public class Address {
      */
     public Address(String address) {
         requireNonNull(address);
-        checkArgument(isValidAddress(address), MESSAGE_CONSTRAINTS);
         value = address;
     }
 
-    /**
-     * Returns true if a given string is a valid email.
-     */
-    public static boolean isValidAddress(String test) {
-        return test.matches(VALIDATION_REGEX);
-    }
-
```

Comment:
Should we instead have an isValidAddress method that can account for nil addresses, instead of removing this altogether?

Reply:
If all inputs are valid addresses, there is no point in having the isValidAddress method